File: src/main/java/com/intellij/tapestry/intellij/view/nodes/ClassNode.java
package com.intellij.tapestry.intellij.view.nodes;

import com.intellij.ide.projectView.PresentationData;
import com.intellij.openapi.module.Module;
import com.intellij.psi.PsiClassOwner;
import com.intellij.util.PlatformIcons;

public class ClassNode extends TapestryNode {

    public ClassNode(PsiClassOwner psiClassOwner, Module module) {
        super(module);

        init(psiClassOwner, new PresentationData(psiClassOwner.getName(), psiClassOwner.getName(), PlatformIcons.CLASS_ICON, null));
    }
}

File: src/main/kotlin/com/github/rar91279/plugin/tapestry/intellij/view/nodes/ClassNode.kt
package com.github.rar91279.plugin.tapestry.intellij.view.nodes

import com.intellij.ide.projectView.PresentationData
import com.intellij.openapi.module.Module
import com.intellij.psi.PsiClassOwner
import com.intellij.util.PlatformIcons

class ClassNode(psiClassOwner: PsiClassOwner, module: Module) : TapestryNode(module) {
    init {
        init(psiClassOwner, PresentationData(psiClassOwner.name, psiClassOwner.name, PlatformIcons.CLASS_ICON, null))
    }
}
